CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,58 @@
+# Code of Conduct
+## Table of contents
+1.	Lawfulness
+2.	Dealing with Data
+3.	Transparency Objectivity and Truth
+4.	Working alone and with others
+
+## Lawfulness
+
+### 1a. Lawful knowledge and adherence
+
+We as the project contributors and maintainers will always act in accordance with the law, developing a full knowledge of, and ensuring compliance with, all relevant regulatory regimes.
+
+### 1b. Privacy and confidentiality protection
+
+We as the project contributors and maintainers have a duty to act so as to protect the privacy and confidentiality of data, respecting the ownership of proprietary data, and in not exposing data (within private or public fora) that might cause any harm to individuals or legal entities.
+
+### 1c. Equality legislation
+
+We as the project contributors and maintainers have a duty not to break gender, race, ethnicity, martial status, religion, belief, disability, or age equality legislation. In particular, such attributes should not place individuals at any disadvantage within models or any automated decisions.
+
+## Dealing with Data
+
+### 2a. Surrogate features and bias
+
+We as the project contributors and maintainers are responsible for detecting and flagging features that might be surrogate to other features that violate fundamental equality rights (gender, race, religion, etc). In general proxy features need to always be checked against social discriminating features (see also 1c. Equality legislation)
+
+### 2b. De-Identification
+
+We as the project contributors and maintainers shall not apply any technique (combination, enriching, etc) to turn information that has been designed to be “de-identifiable” into “identifiable” again.
+
+## Transparency Objectivity and Truth
+
+### 3a. Transparency as a duty
+
+We as the project contributors and maintainers will strive for transparency within as wide a forum as allowable by legal and proprietary constraints. The data scientist will not withhold concerns or potential limitations form colleagues.
```

Comment:
Super small typo, looks like it should be "from" instead of "form"

Reply:
Thank you! It's done.